docs/GettingStarted.md

```diff
@@ -108,27 +108,32 @@ The following steps need to be executed *once* before the first build on a new c
 
 ##### Setting compile time options when building from Developer Command Prompt
 
-To build with specific compile time options, append `/p:DefineConstants=<option_name>`. Options available include:
+To build with the specific compile time options for disabling JIT compiler and/or the Interpreter, append "`/p:<option>=True`". Available options are:
 
-1. `CONFIG_BPF_JIT_ALWAYS_ON` - Compile eBPF Execution Context without support for eBPF interpreter.
```

Comment:
The flag CONFIG_BPF_JIT_ALWAYS_ON was intentionally chosen in https://github.com/microsoft/ebpf-for-windows/issues/737 to match the Linux define to ease cross-platform understanding and documentation.

I guess it's not strictly required to keep it but it would be nice if the documentation somewhere gave a comparison to it on Linux.

Reply:
Clear. The change to `CONFIG_BPF_INTERPRETER_DISABLED` is to better clarify the *actual* action of the option (i.e. disable the interpreter), now that we can separately control the JIT switch.
I've put a note on that build option, recalling the affinity with the Linux switch.